src/main/java/org/cloudfoundry/promregator/config/Target.kt

```diff
@@ -0,0 +1,41 @@
+package org.cloudfoundry.promregator.config
+
+import mu.KotlinLogging
+import java.util.regex.Pattern
+import java.util.regex.PatternSyntaxException
+
+private val logger = KotlinLogging.logger { }
+
+data class Target(
+        val api: String = "__default__" ,
```

Comment:
You are using the constant `__default__` several times. Though I don't have a problem with it in general, I don't like that it is a literal. Can we make a constant out of it somehow?

Reply:
Yes. I'll make it a constant